### PR TITLE
PR For #24: text alignment

### PR DESCRIPTION
**Pull request by MosesStewart**
In this issue we . . .

In this issue we will . . .

- Updated the pp-plot legend to from `Bootstrap Replicates` -> `Bootstrap Reps`
- Standardized the alignment of the textbox in both the pp-plot and crossings plots

Adding @jmshapir and @rcalvo12

### CODE REVIEW

File: src/BootstrapReport/main.py
Comment (jmshapir) [inline src/BootstrapReport/main.py:218]:
```suggestion
                    c = plt_set['pointcolor'], label='Bootstrap reps.')
```

**Review by jmshapir — CHANGES_REQUESTED** [review]
@MosesStewart thanks!

I had one small note.

It would be great if you can paste an example plot into the pull thread here so we can make sure the aesthetics look right.

Thanks!

**Comment by MosesStewart** [comment]
ppplottest2

@jmshapir

File: src/BootstrapReport/main.py
Comment (jmshapir) [inline src/BootstrapReport/main.py:223]: @MosesStewart is it possible to left-justify the text in the lower right box? I think that was the previous default; now it seems to be right-justified.

Thanks!

image

Comment (MosesStewart) [reply src/BootstrapReport/main.py:223]: @jmshapir the text gets justified to the same side as where we decide to place the box, which was why we had it the other way before. 

The only way I could think of accomplishing both left-justified text and a right-aligned text box would be to set it to left aligned, and have a series of `if` - `then` statements counting how many digits thee `num. replicates` are then changing the coordinates accordingly. If this sounds alright then I can implement.

Comment (jmshapir) [reply src/BootstrapReport/main.py:223]: @MosesStewart thanks!

In that case it seems cleaner to leave it right-justified.

@rcalvo12 do you agree?

Comment (rcalvo12) [reply src/BootstrapReport/main.py:223]: @MosesStewart I think there might be a way to achieve what @jmshapir is looking for. See [this stack exchange thread](https://stackoverflow.com/questions/25771752/position-text-box-fixed-in-corner-and-correctly-aligned) which references [this function](https://matplotlib.org/stable/api/offsetbox_api.html) within matplotlib. However, if this proves tricky to implement, I agree with @jmshapir that leaving it as right justified would be fine.

File: src/BootstrapReport/helpers.py
Comment (rcalvo12) [inline src/BootstrapReport/helpers.py:275]: @MosesStewart, in order for the box to be equal distance from the bottom and side we need to use either (0.955, 0.045) or (0.945, 0.055). I think (0.955, 0.045) is slightly preferable since it gives more space for the replicates.

**Review by rcalvo12 — CHANGES_REQUESTED** [review]
@MosesStewart Thanks for your work here! I left one note and responded to the thread started by @jmshapir.

File: src/BootstrapReport/helpers.py
Comment (rcalvo12) [reply src/BootstrapReport/helpers.py]: @MosesStewart, I implemented the change in https://github.com/JMSLab/BootstrapReport/pull/25/commits/7a4342aa5431996b54ed2b5f85089835cd32f1c0, resolving.

File: src/BootstrapReport/helpers.py
Comment (rcalvo12) [reply src/BootstrapReport/helpers.py:275]: @jmshapir, from gamma_example:
image

Comment (jmshapir) [reply src/BootstrapReport/helpers.py:275]: Looks good thanks!

File: src/BootstrapReport/main.py
Comment (MosesStewart) [reply src/BootstrapReport/main.py]: @rcalvo12 thank you for sharing! I had no idea that the class `offsetbox` existed within Matlab!

I implemented a working version below:

ppplottest2

There were some limitations, in that I could not adjust the transparency of the box, and I could not get the box to have rounded corners, so I changed the legend box to look the same as the `offsetbox`. If this looks okay then I can commit!

Comment (rcalvo12) [reply src/BootstrapReport/main.py]: @MosesStewart awesome, I think you can adjust some of those things by using a syntax like this:
```
        ob = offsetbox.AnchoredText(plot_data, loc='lower right')
        ob.patch.set(boxstyle = 'round, pad = 0.75, rounding_size = 0.3', facecolor = 'white', alpha = 0.86)
```

Comment (MosesStewart) [reply src/BootstrapReport/main.py]: @rcalvo12 thanks implemented in c8ce870

Comment (MosesStewart) [reply src/BootstrapReport/main.py]: ppplottest2

**Review by jmshapir — APPROVED** [review]
Looks good to me!

Before we merge let's wait for @rcalvo12 to approve and for Actions to finish testing.

**Review by rcalvo12 — APPROVED** [review]
Thanks @MosesStewart!

**Pull request merged**
